Dockerfile

```diff
@@ -1,62 +1,37 @@
-FROM openanalytics/r-base
+FROM openanalytics/r-ver:4.0.5
 
 MAINTAINER Stijn Van Hoey stijn.vanhoey@inbo.be 
 
-# system libraries of general use
-RUN apt-get update && apt-get install -y \
-    sudo \
-    pandoc \
-    pandoc-citeproc \
-    libcurl4-gnutls-dev \
-    libcairo2-dev \
-    libxt-dev \
-    libssl-dev \
-    libssh2-1-dev \
-    libssl1.0 \
+RUN apt-get update && apt-get install -y --no-install-recommends \
     gdal-bin \
-    libgdal-dev \
-    libproj-dev \
-    libgeos-dev \
-    libudunits2-dev \
-	libgit2-dev
-	
-# Dependencies for rgdal and rgeos
-RUN  apt-get update && apt-get install -y software-properties-common && \
-     apt install gdal-bin
-	 
-# Dependencies for devtools
-RUN R -e "install.packages(c('gert', 'usethis'), repos='https://cloud.r-project.org/')"	 
-
-# install imports of reporting-grofwild app that are not on cloud
-RUN R -e "install.packages(c('shiny', 'sp', 'plyr', 'devtools', 'methods', 'reshape2', 'mgcv', 'rgdal', 'rgeos', 'raster', 'stringr'), repos = 'https://cloud.r-project.org/')"
-RUN R -e "devtools::install_version('plotly', version = '4.9.2.1', repos = 'http://cran.us.r-project.org')"
-RUN R -e "devtools::install_github('inbo/INBOtheme')"
-RUN R -e "install.packages('https://cran.r-project.org/src/contrib/Archive/DT/DT_0.12.tar.gz', repos = NULL, type = 'source')"
-RUN R -e "install.packages(c('flexdashboard'), repos='https://cloud.r-project.org/')"
-
-# install depends of reporting-grofwild app
-RUN R -e "install.packages(c('maptools'), repos='https://cloud.r-project.org/')"
-
-# install suggests of reporting-grofwild app
-RUN R -e "install.packages(c('leaflet', 'mapview'), repos='https://cloud.r-project.org/')"
+    libproj15 \
+    libgeos-3.8.0 libgeos-c1v5  \
+    curl \ 
+    ca-certificates \
+    pandoc \
+    libudunits2-0 \
+    && rm -rf /var/lib/apt/lists/*
+
+# Use the remotes package instead of devtools as it is mutch lighter
+RUN R -q -e "install.packages('remotes')"
```

Comment:
I now get this error :
```
 => ERROR [ 4/11] RUN R -q -e "remotes::install_cran(c('shiny', 'sp', 'plyr', 'reshape2', 'mgcv', 'rgdal', 'rgeos  1.0s
------
 > [ 4/11] RUN R -q -e "remotes::install_cran(c('shiny', 'sp', 'plyr', 'reshape2', 'mgcv', 'rgdal', 'rgeos', 'raster', 'stringr', 'maptools', 'leaflet', 'mapview', 'flexdashboard'))":
#8 0.879 > remotes::install_cran(c('shiny', 'sp', 'plyr', 'reshape2', 'mgcv', 'rgdal', 'rgeos', 'raster', 'stringr', 'maptools', 'leaflet', 'mapview', 'flexdashboard'))
#8 0.880 Error in loadNamespace(name) : there is no package called ‘remotes’
#8 0.880 Calls: :: ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
#8 0.880 Execution halted
```

Reply:
The error remains. But when building with the extended docker build command (see [docker_build_log.txt](https://github.com/inbo/reporting-rshiny-grofwildjacht/files/8831202/docker_build_log.txt)) it seems to "work", as I didn't see any errors. But when trying to run the app using this command: `docker run -p 3845:3838 openanalytics/r-base R -e 'reportingGrofwild::runWildApp(public=TRUE)'` I get the following error:
```
> reportingGrofwild::runWildApp(public=TRUE)
Error in loadNamespace(name) :
  there is no package called ‘reportingGrofwild’
Calls: :: ... loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
Execution halted
```

Reply:
UPDATE: By using the docker desktop app on my laptop I was able to get the public app running.

Reply:
still no idea how to get private part running

Reply:
If you build the image using `docker build --progress=plain --no-cache -t openanalytics/wildapp .`, you will need to use `openanalytics/wildapp` as image name in the run command: `docker run -p 3845:3838 openanalytics/wildapp R -e 'reportingGrofwild::runWildApp(public=TRUE)'` (and thus `docker run -p 3845:3838 openanalytics/wildapp R -e 'reportingGrofwild::runWildApp(public=FALSE)` in case of the private part)

Reply:
This works!

Thanks